### PR TITLE
PG-1012 Documented set up of the_heap access method

### PR DESCRIPTION
**Pull request by nastena1606**
Added How to decrypt a table

### CODE REVIEW

**Review by janwieremjewicz — APPROVED** [review]
LGTM

**Review by ImTheKai — APPROVED** [review]

File: documentation/docs/setup.md [hunk 1]
```diff
@@ -72,6 +76,65 @@ Load the `pg_tde` at the start time. The extension requires additional shared me
...
+       sudo systemctl restart postgresql.service
+       ```
+    
+    * On RHEL and derivatives
+
+       ```sh
+       sudo systemctl restart postgresql-17
+       ```
+
+    After you enabled `pg_tde` and started the Percona Server for PostgreSQL, a principal key and a keyring for WAL are created. We highly recommend you to create your own keyring and rotate the principal key. This is because the default principal key is created from the local keyfile and is stored unencrypted.
```
Comment (dAdAbird) [inline documentation/docs/setup.md:103]: In fact, right now we set up global keys on the first server start in any case (even when pg_tde.wal_encrypt is off)

Comment (nastena1606) [reply documentation/docs/setup.md]: Reworked

File: documentation/docs/setup.md [hunk 2]
```diff
@@ -72,6 +76,65 @@ Load the `pg_tde` at the start time. The extension requires additional shared me
...
+
+Perform this step if you [installed Percona Server for PostgreSQL :octicons-link-external-16:](https://docs.percona.com/postgresql/17/installing.html). Otherwise, proceed to the [Next steps](#next-steps).
+
+1. Enable WAL level encryption using the ALTER SYSTEM SET command:
+
+    ```sql
+    ALTER SYSTEM set pg_tde.wal_encrypt = on;
+    ```
+
+2. Restart the server to apply the changes.
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply documentation/docs/setup.md:91]: Let's have @shahidullah79's confirmation on that

**Pull request merged**